### DISCUSSION TITLE
关于新的独立vue web项目如何应用于lamp-cloud生态体系的疑问

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#206] [02.提问]
如果现在有个新的独立web前端，这个前端可能是一个新的业务项目，要访问lamp-cloud的相应接口和后增的业务接口，这个独立前端的接口调用和权限认证这些必须跟lamp-web保持一致嘛？ 如果不是一致的 需要怎么调用才能实现它的权限也能收到lamp-cloud该架构体系的权限和多租户的管控？

**Comment by zuihou** [comment]
重新实现一遍lamp-web实现的  前端按钮权限，前端动态路由，请求头中传递Token、Tenant、authorization 等参数

**Reply by zqb666** [reply]
能出一下这块的 demo和文档吗
